blockchain/txvalidator.go

```diff
@@ -220,6 +222,11 @@ func CheckTransactionOutput(version uint32, txn *Transaction) error {
 			if !CheckOutputProgramHash(output.ProgramHash) {
 				return errors.New("output address is invalid")
 			}
+			// output value must >= 0
```

Comment:
The output value check should not put at here, it should be checked in any condition.

Reply:
fixed